ai-platform-unified/notebooks/unofficial/model_monitoring/model_monitoring.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>This is hard-coded, which is strange.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/318/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I wanted people to see a prototypical working example without having to copy that bucket from an email notif, but I'll see if I can mine that bucket name from the env via gcloud or the sdk.</p>